src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -478,17 +507,27 @@ def main():
     conf = None
 
     try:
+        # setup container.conf file for the callback server logs if needed
+        if loader_helper.is_config_modification_required():
+            if click.confirm(f"Set up a containers.conf file at {loader_common_names.CONTAINERS_CONF_PATH} if does not exist already?\n"
+                             f"Params 'seccomp_profile' and 'log_driver' will be added/updated under section [containers]"):
+                loader_helper.setup_callback_server_logs()
+            else:
+                print("Permission denied and exiting ...")
+                return
+
         # start podman service
         proc = loader_helper.start_podman_service(uid)
 
-        # set up conf, start callback server, and upload assemblies to workspace
+        # set up conf for uploader, start callback server, and upload assemblies to workspace
         conf = Conf(
-            job_dir,
-            output_dir,
-            _process_input,
-            kb_base_url,
-            token_filepath,
-            workers,
+            job_dir=job_dir,
+            output_dir=output_dir,
+            kb_base_url=kb_base_url,
```

Comment:
Does it need to pass this variable since it's creating its own ws client?

Reply:
Updated!

Reply:
I was talking about the `kb_base_url` param, but if `output_dir` can be dropped too great

Reply:
cannot drop kb_base_url. `kb_base_url` is one of envs that get passed into the callback server. 
https://github.com/kbase/collections/blob/e14b49d2490d734b6e9b35f8764eea0f62cadca7/src/loaders/common/callback_server_wrapper.py#L146

Reply:
:+1: